recipe/meta.yaml

```
@@ -24,16 +24,17 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - versioneer
```

https://github.com/pymc-devs/pymc/blob/v5.23.0/pyproject.toml#L2
versioneer[toml]==0.29
```suggestion
    - versioneer 0.29
    - toml
```